src/content/getusermedia/exposure/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<!--
+ *  Copyright (c) 2020 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+-->
+<html>
+<head>
+
+    <meta charset="utf-8">
+    <meta name="description" content="WebRTC code samples">
+    <meta name="viewport" content="width=device-width, user-scalable=yes, initial-scale=1, maximum-scale=1">
+    <meta itemprop="description" content="Client-side WebRTC code samples">
+    <meta itemprop="image" content="../../../images/webrtc-icon-192x192.png">
+    <meta itemprop="name" content="WebRTC code samples">
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta id="theme-color" name="theme-color" content="#ffffff">
+
+    <base target="_blank">
+
+    <title>Control Exposure</title>
+
+    <link rel="icon" sizes="192x192" href="../../../images/webrtc-icon-192x192.png">
+    <link href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="../../../css/main.css">
+
+    <style>
+        div.label {
+            display: inline-block;
+            font-weight: 400;
+            margin: 0 0.5em 0 0;
+            width: 10em;
+        }
+    </style>
+</head>
+
+<body>
+
+<div id="container">
+    <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC samples</a>
+        <span>Control Exposure</span></h1>
+
+    <video id="gum-local" autoplay playsinline></video>
+    <button id="showVideo">Open camera</button>
+
+    <div>
+        <div class="label">Exposure Mode:</div>
+        <select name="exposureMode" id="exposureMode" disabled>
+        </select>
+    </div>
+
+    <div>
+        <div class="label">Exposure Time:</div>
+        <input name="exposureTime" type="range" disabled>
+    </div>
+	
+    <div>
+        <div class="label">Exposure Compensation:</div>
+        <input name="exposureCompensation" type="range" disabled>
+    </div>
+	
+	<div>
+        <div class="label">Brightness:</div>
+        <input name="brightness" type="range" disabled>
+    </div>
+	
+	<div>
+        <div class="label">White Balance Mode:</div>
+        <select name="whiteBalanceMode" id="whiteBalanceMode" disabled>
```

Comment:
I realized that `whiteBalanceMode` is out of scope. Should I remove it @fippo ?

Reply:
the more knobs the merrier